crane/container.go

```diff
@@ -51,16 +51,17 @@ type ContainerInfo interface {
 }
 
 type container struct {
-	id        string
-	RawName   string
-	RawUnique bool            `json:"unique" yaml:"unique"`
-	RawImage  string          `json:"image" yaml:"image"`
-	RawBuild  BuildParameters `json:"build" yaml:"build"`
-	RawRun    RunParameters   `json:"run" yaml:"run"`
-	RawRm     RmParameters    `json:"rm" yaml:"rm"`
-	RawStart  StartParameters `json:"start" yaml:"start"`
-	RawExec   ExecParameters  `json:"exec" yaml:"exec"`
-	hooks     hooks
+	id          string
+	RawName     string
+	RawUnique   bool            `json:"unique" yaml:"unique"`
+	RawImage    string          `json:"image" yaml:"image"`
+	RawRequires []string        `json:"requires" yaml:"requires"`
```

Comment:
did you consider having this somewhere else? I can't find of a better place, but it does feel strange there as most of the other values are first-class Docker citizens